canonical-web-and-design/snapcraft.io
# Set and remove default tracks

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/1966
Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/1967

**NOTE: You will need a snap with tracks to QA this. Your best option is to set your local env to use staging and I'll add you as a collaborator on a snap I have with tracks.**

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/snap_name/releases
- Change track from latest
- Click the "Set as default track" button
- A modal should appear with some text explaining what's going to happen
- Click "Set {track} as default track"
- The default track should be updated (you should see a notification)
- Reload the page to ensure the change has taken effect
- Click "Clear default track"
- Follow similar steps as before
- Do some more clicking and setting/ unsetting
